concordium-consensus/src/Concordium/External.hs

```diff
@@ -891,14 +891,17 @@ decodeBlockHash blockcstr = readMaybe <$> peekCString blockcstr
 decodeAccountAddress :: CString -> IO (Either String AccountAddress)
 decodeAccountAddress acctstr = addressFromBytes <$> BS.packCString acctstr
 
--- |Decode a null-terminated string as either an account address (base-58) or a
+-- |Decode a null-terminated string as either an account address (base-58), account index (AccountIndex) or a
 -- credential registration ID (base-16).
-decodeAccountAddressOrCredId :: CString -> IO (Maybe (Either CredentialRegistrationID AccountAddress))
```

Comment:
I don't think this function is needed anymore so should be removed.

Reply:
It was actually, just did not push it, was waiting for the merge of the change in `concordium-base`.